package.json

```diff
@@ -16,7 +16,7 @@
     "test": "npm run lint && npm run test-only",
     "test-only": "mocha -c --require test/setup.js --recursive ./test",
     "test-app": "./bin/nodejs-dashboard.js node test/app/index.js",
```

Comment:
@ryan-roemer Should/Could it be done as:

"test-app": "node bin/nodejs-dashboard.js -- node test/app/index.js",